stories/models.py

```diff
@@ -90,6 +131,10 @@ def __str__(self):
         return f"{self.title} ({self.id})"
 
 
+# Agent types
+SUSPECT, WITNESS, ENVIRONMENT = "SUSPECT", "WITNESS", "ENVIRONMENT"
```

Comment:
Probably, it can be implemented not as cringy as the way it is, but I don't know how.